client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -149,9 +150,9 @@ export default function getThankYouPageUrl( {
 		// extract a product from the cart, in siteless checkout there should only be one
 		const productSlug = cart?.products[ 0 ]?.product_slug;
 
-		const thankYouUrlSiteLess = `/checkout/jetpack/thank-you/no-site/${
-			productSlug ?? 'no_product'
-		}`;
+		const thankYouUrl = isEnabled( 'jetpack/user-licensing' )
+			? `/checkout/jetpack/thank-you/licensing-auto-activate/${ productSlug ?? 'no_product' }`
```

Comment:
Could you make sure we add a test for this new URL to `client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js`?